packages/core/src/next.ts

```diff
@@ -0,0 +1 @@
+export * from "./components/next";
```

Comment:
Why this again?

Reply:
yet

Reply:
Feels a bit like preparation for AC but OK :)

Reply:
I removed the preparation for AC, you are right 🙃